capstone-frontend/src/components/UserCard.jsx

```diff
@@ -7,36 +7,78 @@ import '../styles/Groups.css';
 class UserCard extends Component {
   removeMember = () => {
     const jsonBody = {
-      "remove_memberIDs": this.props.user.id, 
+      "remove_memberIDs": this.props.user.id,
       "id": this.props.club.id
     }
-    fetch(`/api/clubs`, {
-        method: "put", 
-        body: JSON.stringify(jsonBody)
-      })
+    fetch(`/api/clubs`, { method: "put", body: JSON.stringify(jsonBody) })
       .then(this.props.removeMember(this.props.user.id))
+      .catch(function (err) {
+        //TODO #61: Centralize error output
+        alert(err);
+      });
+  }
+
+  acceptMember = () => {
+    const jsonBody = {
+      "remove_requestIDs": this.props.user.id,
+      "add_memberIDs": this.props.user.id,
+      "id": this.props.club.id
+    };
+    fetch("/api/clubs", { method: "put", body: JSON.stringify(jsonBody) })
+      .then(this.props.fetchData)
+      .catch(function (err) {
+        //TODO #61: Centralize error output
+        alert(err);
+      });
+  }
+
+  rejectMember = () => {
+    const jsonBody = {
+      "remove_requestIDs": this.props.user.id,
+      "id": this.props.club.id
+    };
+    fetch("/api/clubs", { method: "put", body: JSON.stringify(jsonBody) })
+      .then(this.props.fetchData)
+      .catch(function (err) {
+        //TODO #61: Centralize error output
+        alert(err);
+      });
   }
 
   addFriend = () => {
     //TODO #86: Create friend functionality
   }
 
   render() {
-    const removeMember = this.props.club 
-                             && this.props.club.ownerID === window.localStorage.getItem("userID")
-                             && this.props.club.ownerID !== this.props.user.id
-                             && <Button id="remove-member" variant="danger" onClick={this.removeMember}>
-                                  Remove&nbsp;Member
-                                </Button>;
+    const isMember = this.props.club && this.props.club.memberIDs && this.props.club.memberIDs.includes(this.props.user.id);
+    const isRequester = this.props.club && !isMember && this.props.club.requestIDs && this.props.club.requestIDs.includes(this.props.user.id);
+    const removeMember = this.props.club
+      && this.props.club.ownerID === window.localStorage.getItem("userID")
+      && this.props.club.ownerID !== this.props.user.id
+      && <Button className="mt-2" cvariant="danger" onClick={this.removeMember}>
```

Comment:
Not sure if this is a typo or not, but I haven't seen `cvariant` before, is it supposed to be `variant`?

Reply:
A typo! Nice catch

Reply:
Strange that no errors were surfaced, and it seemed like the button had the right style anyways.